spec/support/shared_examples/an_entity_mapper_missing_entity_destroy_operation.rb

```diff
@@ -23,17 +18,17 @@
       allow(TradeTariffBackend).to receive(:handle_soft_deletes?).and_return(false)
     end
 
-    it "does not hide the missing xml nodes from the #{relation} view" do
-      before_import_primary_keys = relation.where(filter).pluck(*model_primary_keys)
-
-      entity_mapper.import
-
-      after_import_primary_keys = relation.where(filter).pluck(*model_primary_keys)
-
-      expect(after_import_primary_keys).to include(*before_import_primary_keys)
+    # rubocop:disable RSpec/MultipleExpectations
+    it "does not hide the missing xml nodes from the #{model} view" do
+      if has_hard_deletes
+        expect { entity_mapper.import }.to change_view_count.by(-1)
+      else
+        expect { entity_mapper.import }.not_to change_view_count
+      end
     end
```

Comment:
it looks this should be split into two different contexts:
with has hard deletes
and without hard deletes

Reply:
Sadly that's not actually possible

Reply:
I'll be getting rid of this in like 2 weeks